server/quizzes/routes.js

```diff
@@ -28,6 +30,17 @@ quizRouter.get('/:id', async (req, res) => {
 //   .catch(error => res.status(500).send(error));
 // });
 
+// Send Score - will change to POST method.
+quizRouter.get('/:user/:id/:score', async (req, res) => {
```

Comment:
What is this route trying to do? Is this trying to get the user's score for a given quiz they have taken?

Reply:
Should this be a POST request? I think rating might be a better name. Perhaps the endpoint should be `users/:user_id/quizzes/:quiz_id/rating/:rating_id`

Reply:
Yeah, it should be a POST request. I will make the modification in the coming days. I like the URL you have proposed. Think we should all have a discussion to get the route name to be somewhat uniform.  Would you kindly approve the merge for now?